org.jacoco.report/src/org/jacoco/report/check/Limit.java

```diff
@@ -80,10 +80,11 @@ public CounterEntity getEntity() {
 
 	/**
 	 * Sets the counter entity to check.
+	 * 
+	 * TODO: use CounterEntity directly once Maven 3 is required.
```

Comment:
Not only this Maven-ism affected API, but also this [`TODO`-comment is visible in documentation](https://www.jacoco.org/jacoco/trunk/doc/api/org/jacoco/report/check/Limit.html#setCounter(java.lang.String)) 😞 And while we already require Maven 3 (#821), I propose to just move this `TODO`-comment outside of javadoc.